test/http.test.js

```diff
@@ -1,16 +1,16 @@
 const OError = require('..')
 const HttpErrors = require('../http')
 
-describe('OError/http', () => {
-  it('is instance of OError', () => {
+describe('OError/http', function () {
```

Comment:
After being a bit confused, looks like this has been the prettier style for a while, and it's our standard linting thats changing it?

Reply:
(Just realised my comment isn't clear: I meant the space before the parens)

Reply:
That seems to be the case for anonymous functions now... even with standard.